### DISCUSSION TITLE
Is version 4.4.5 of UTM-HV a slow version？

### DISCUSSION THREAD

**Discussion by wwechoww** [utmapp/UTM#5958] [UTM Help (iOS)] [answered]
Sorry, I am a beginner in this area.

When I install version 4.4.5 of UTM-HV using the Troll Store on my iPad, the UTM icon shows UTM SE. Is this a slow version?

I have downloaded version 4.2.5 of UTM-HV before, and its icon only shows UTM.

**Comment by osy** [comment]
Make sure you downloaded UTM-HV.ipa

**Reply by R3tr0ooo** [reply]
I ran into the same issue. I am 1000% sure that I installed "UTM-HV.ipa", but it's still showing up "UTM SE" after I installed it. It's not reporting any errors to run an emulated VM, but I'm not sure if it's using JIT. iPhone Xs Max, iOS 16.6.1, installed via TrollStore 2.0.8.

**Reply by osy** [reply]
https://github.com/utmapp/UTM/pull/5968

**Comment by osy** [comment, marked as answer]
https://github.com/utmapp/UTM/pull/5968
